### PR TITLE
Fix/array input l10n

### PR DESCRIPTION
**Pull request by abea**
Fixes stray l10n string

### CODE REVIEW

**Comment by abea** [comment]
Not meant to merge until next release.

File: modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default {
   },
   computed: {
     editLabel () {
-      return `Edit ${this.field.label}`;
+      return `Edit ${this.$t(this.field.label)}`;
```
Comment (boutell) [inline modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue:43]: The whole thing needs l10n! See how other strings with interpolation are set up.

**Review by boutell — CHANGES_REQUESTED** [review]

File: modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue (hunk 1 above)
Comment (boutell) [reply modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue:43]: i.e. no use if we don't l10n "Edit"

Comment (abea) [reply modules/@apostrophecms/schema/ui/apos/components/AposInputArray.vue:43]: oh roit

**Review by boutell — APPROVED** [review]

**Pull request merged**